sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py

```diff
@@ -40,7 +40,9 @@ class CosmosClient:
     Use this client to configure and execute requests to the Azure Cosmos DB service.
     """
 
-    def __init__(self, url, auth, consistency_level="Session", connection_policy=None):
+    def __init__(
+        self, url, auth, consistency_level="Session", connection_policy=None
+    ):  # pylint: disable=missing-client-constructor-parameter-credentials,missing-client-constructor-parameter-kwargs,line-too-long
```

Comment:
Just want to catch this so it's not a problem later - should be: disable=missing-client-constructor-parameter-**credential**. We had a discussion today in the office and decided that the credential param should be singular, not plural.

Reply:
@kristapratico thank you I will fix this before merging